src/nni_manager/core/ipcInterface.ts

```diff
@@ -22,13 +22,22 @@ const ipcIncomingFd: number = 4;
  * @returns binary command data
  */
 function encodeCommand(commandType: string, content: string): Buffer {
-    const contentBuffer: Buffer = Buffer.from(content);
-    if (contentBuffer.length >= 1_000_000) {
-        throw new RangeError('Command too long');
+    let contentBuffer: Buffer = Buffer.from(content);
```

Comment:
suggest to add docstring to explain the protocol

Reply:
The doc will be added to wiki.